people.json

```diff
@@ -12465,6 +12465,23 @@
         "slack_id": "",
         "image": "jan-laudahn.jpg"
     },
+    {
+        "name": "Jan Melén",
+        "bio": "Jan Melén is an expert in networking, open-source software, and cloud technologies, with a career spanning over two decades. He is a staunch advocate for Free and Open-Source Software (FOSS) adoption within Ericsson and the broader community he engages with.</p>\n\n<p>Jan has served as a Research Leader for Cloud Research, driving innovations in cloud computing. Since 2019, he has led a team dedicated to CNCF open-source projects, contributing significantly to Ericsson's Kubernetes distribution, and fostering an "upstream-first" culture within the company. He now oversees Ericsson’s open-source contributions and coordinates collaborations with CNCF projects. With his technical expertise and leadership, Jan aims to support CNCF's mission and strengthen the global cloud-native ecosystem.",
```

Comment:
The quotes around "upstream-first" are causing the json to be invalid. I think you'll need to escape them using a "\" before each of them...